internal/cluster/distribution/eks/node_pool.go

```diff
@@ -32,6 +32,7 @@ type NewNodePool struct {
 		MinSize int  `mapstructure:"minSize"`
 		MaxSize int  `mapstructure:"maxSize"`
 	} `mapstructure:"autoscaling"`
+	VolumeSize   int
```

Comment:
I'm not sure it is an issue but the `mapstructure` tag is missing here.

Reply:
[Fixed](https://github.com/banzaicloud/pipeline/compare/4d212471fe21678f970c77b6503092ae1ba18f1c..9622b4c655c2b68e5644d44e1cfa83a60d7b9cb4).
It wasn't an issue during tests, because `mapstructure` handles casing differences insensitively as long as the field is exported, but I decided to fix it for consistency and aesthetical reasons.